### PR TITLE
python-pyswarm: add new package

### PR DESCRIPTION
**Pull request by 3rav**

### CODE REVIEW

File: mingw-w64-python-pyswarm/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32' 'clangarm64')
+url="https://pyswarms.readthedocs.io/en/latest/"
+depends=("${MINGW_PACKAGE_PREFIX}-python"
+         "${MINGW_PACKAGE_PREFIX}-python-numpy"
+         "${MINGW_PACKAGE_PREFIX}-python-matplotlib")
+makedepends=("${MINGW_PACKAGE_PREFIX}-python-build"
+             "${MINGW_PACKAGE_PREFIX}-python-installer"
+             "${MINGW_PACKAGE_PREFIX}-python-setuptools"
+             "${MINGW_PACKAGE_PREFIX}-python-wheel")
+license=('spdx:BSD')
```
Comment (Biswa96) [inline mingw-w64-python-pyswarm/PKGBUILD:19]: The project is using MIT license, see https://github.com/ljvmiranda921/pyswarms

File: mingw-w64-python-pyswarm/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+source=("https://files.pythonhosted.org/packages/79/1e/254c108b5e65c65d57a83a9a448405ea8b6a6c5c10dada8bcab4e9d9a831/${_realname}-${pkgver}.tar.gz")
```
Comment (Biswa96) [inline mingw-w64-python-pyswarm/PKGBUILD:20]:
```suggestion
source=("https://pypi.org/packages/source/${_realname::1}/${_realname}/${_realname}-${pkgver}.tar.gz")
```

File: mingw-w64-python-pyswarm/PKGBUILD (hunk 1 above)
Comment (3rav) [reply mingw-w64-python-pyswarm/PKGBUILD:19]: No, this is pyswarm not pyswarms, see https://pythonhosted.org/pyswarm/

Comment (Biswa96) [reply mingw-w64-python-pyswarm/PKGBUILD:19]: The last commit was in 2015. Should this old package be added?

Comment (3rav) [reply mingw-w64-python-pyswarm/PKGBUILD:19]: This is an optimization script, I think that nothing much will change in it in the future, but it is useful.

**Pull request merged**